src/hclib-runtime.c

```diff
@@ -216,13 +168,8 @@ void hclib_entrypoint() {
      */
     hclib_global_init();
 
-#ifdef HC_COMM_WORKER
-    const int have_comm_worker = 1;
-#else
-    const int have_comm_worker = 0;
-#endif
     // init timer stats
-    hclib_initStats(hclib_context->nworkers, have_comm_worker);
```

Comment:
Looks like we can delete this second parameter too, and the static have_comm_worker in hclib-timer.c